repo/fsrepo/fsrepo.go

```diff
@@ -578,6 +581,26 @@ func (r *FSRepo) GetStorageUsage() (uint64, error) {
 	return du, err
 }
 
+func (r *FSRepo) SwarmKey() ([]byte, error) {
```

Comment:
It could be part of keystore but as keystore isn't scheduled yet it will have to work for now. In future we can migrate it into keystore.